scripts/pi-hole/js/db_graph.js

```diff
@@ -131,54 +160,73 @@ function updateQueriesOverTime() {
     });
 }
 
+Date.prototype.AddInterval = function () {
```

Comment:
Any reason you are extending the Date prototype?

Reply:
I use this some lines further down. I could have done it in many ways, however, this extension seemed a very natural implementation to me.